persistence/persistence-api/src/main/resources/applicationContext-rediscache.xml

```diff
@@ -9,63 +9,40 @@
 
-    <bean id="redisCacheManager" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
-        <property name="targetObject" ref="customRedisCacheProvider" />
-        <property name="targetMethod" value="getCacheManager" />
-        <property name="arguments">
-            <list><ref bean="redissonClient"/></list>
-        </property>
-    </bean>
+        <bean id="redissonClient" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+            <property name="targetObject" ref="customRedisCacheProvider" />
+            <property name="targetMethod" value="getRedissionClient" />
+        </bean>
 
-    <bean id="generalRepositoryCacheResolver" class="org.springframework.cache.interceptor.NamedCacheResolver">
-        <constructor-arg index="0" ref="redisCacheManager"/>
-        <constructor-arg index="1" value="${app.name:cbioportal}GeneralRepositoryCache"/>
-    </bean>
+        <bean id="redisCacheManager" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+            <property name="targetObject" ref="customRedisCacheProvider" />
+            <property name="targetMethod" value="getCacheManager" />
+            <property name="arguments">
+                <list><ref bean="redissonClient"/></list>
+            </property>
+        </bean>
 
-    <bean id="staticRepositoryCacheOneResolver" class="org.springframework.cache.interceptor.NamedCacheResolver">
-        <constructor-arg index="0" ref="redisCacheManager"/>
-        <constructor-arg index="1" value="${app.name:cbioportal}StaticRepositoryCacheOne"/>
-    </bean>
+        <bean id="generalRepositoryCacheResolver" class="org.springframework.cache.interceptor.NamedCacheResolver">
+            <constructor-arg index="0" ref="redisCacheManager"/>
+            <constructor-arg index="1" value="${app.name:cbioportal}GeneralRepositoryCache"/>
+        </bean>
 
-<!--
-    Note : the following bean is missing arguments to the constructor
-    <bean id="cacheManager" class="org.redisson.spring.cache.RedissonSpringCacheManager">
-        <property name="cacheManager" ref="getCacheManager"/>
-    </bean>
+        <bean id="staticRepositoryCacheOneResolver" class="org.springframework.cache.interceptor.NamedCacheResolver">
+            <constructor-arg index="0" ref="redisCacheManager"/>
+            <constructor-arg index="1" value="${app.name:cbioportal}StaticRepositoryCacheOne"/>
+        </bean>
 
-    <bean id="springCacheManager" class="org.springframework.cache.CacheManager">
-        <property name="cacheManager" ref="getCacheManager"/>
-    </bean>
-
-    <bean id="cacheManager" class="org.springframework.cache.jcache.JCacheCacheManager">
-        <property name="cacheManager" ref="getCacheManager"/>
-    </bean>
-
--->
     <bean id="customKeyGenerator" class="org.cbioportal.persistence.util.CustomKeyGenerator"/>
 
-    <!--bean id="ehCacheStatistics" class="org.cbioportal.persistence.util.EhCacheStatistics">
```

Comment:
we need to work out the wiring of a cache statistics service (maybe a stub) for redis